app/views/internal/feedback_messages/_feedback_message.html.erb

```diff
@@ -92,23 +92,23 @@
             </ul>
             <div class="tab-content my-3">
               <div role="tabcard" class="tab-pane fade active show" data-id="<%= feedback_message.id %>" data-userType="reporter" id="reporter-<%= feedback_message.id %>">
-                <h3>Send to:</h3>
+                <h3>Send To:</h3>
```

Comment:
Maybe instead of doing this manually we could add 

```css
text-transform: capitalize;
```

to the relevant CSS?

Reply:
I prefer not to manipulate text with CSS when possible. If this was user-generated content, I think that'd make since, but in this case I think this is more clear.